core-web/src/main/java/org/silverpeas/core/webapi/publication/PublicationResource.java

```diff
@@ -23,6 +23,14 @@
  */
 package org.silverpeas.core.webapi.publication;
 
+import org.silverpeas.core.contribution.publication.model.CompletePublication;
+import org.silverpeas.core.contribution.publication.model.Link;
+import org.silverpeas.core.contribution.publication.model.PublicationDetail;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.silverpeas.core.contribution.publication.model.PublicationDetail'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)